test/RootStateSchema.json

```diff
@@ -813,6 +813,12 @@
             ],
             "type": "object"
         },
+        "FiatConnectSchemaCountryOverrides<FiatAccountSchema.AccountNumber>": {
+            "additionalProperties": {
+                "$ref": "#/definitions/{[PropertyinkeyofPartial<FiatAccountSchemas[T]>]:{regex?:string|undefined;errorString?:string|undefined;errorParams?:Record<string,any>|undefined;};}"
```

Comment:
now that you mention it, i'm also a little baffled by why line 816 is specific `FiatConnectSchemaCountryOverrides<FiatAccountSchema.AccountNumber>` but line 818 is generic :/ because it's generic here, the value contains only the common properties for all `T`. if the `FiatConnectSchemaCountryOverrides` definition were changed to the specific `[Property in keyof Partial<FiatAccountSchemas[FiatAccountSchema.AccountNumber]>]` then line 818 will change to what we expect (i just did this as a sanity check) - not sure if it's worth digging into this, i doubt this will cause you any real problems. since this file doesn't have anything to do with runtime, and typescript doesn't complain if you add more properties than the type asks for so shouldn't be a problem if you need to do migrations on this at a later time